### ISSUE TITLE
error installing node-openalpr on Win10 platform 

### ISSUE THREAD

**Issue by knorlander** [netPark/node-openalpr#10]
Error occurs when running "npm install node-openalpr" from command line.  I also tried loading module from within MS VS 2015 and get same error.  I am relatively new to Node and NPM, but I have verified latest releases of node/npm.    debug.log file attached.  
[npm-debug.log.txt](https://github.com/netPark/node-openalpr/files/120565/npm-debug.log.txt)

I continue to research what the problem is, but am running out of paths to success. thanks

**Comment by kjlaw89** [comment]
Yeah, I couldn't get OpenALPR to build on Windows, so I was unable to update the OpenALPR library to the latest release like the Linux and OSX releases. You can probably get it to work if you install an older version of node-openalpr, 1.0.9 should be safe on Windows.
